Makefile

```diff
@@ -37,13 +42,16 @@ test-w: ${NODE}
 build-module: src/*
 
 serve:
-	$(BABEL_NODE) server.js
+	open dist/index.html
```

Comment:
:-/ Sorry, we'll need to branch on linux to run `google-chrome` or figure out something better, like script/browser or some such thing...